src/pages/Home.tsx

```diff
@@ -17,6 +18,13 @@ const Home: React.FC = () => {
     checked ? document.body.classList.add('dark') : document.body.classList.remove('dark')
   }
 
+  const onPhoneChange = (e: CustomEvent<InputChangeEventDetail>) => {
+    const dialPhone = e.detail.value ?? ''
+    const dialCode = COUNTRY_CODES.find(x => dialPhone.startsWith(x.dialCode))?.dialCode
```

Comment:
I think we should enable the input without the `+` sign too. What if like this:

<img width="473" alt="image" src="https://user-images.githubusercontent.com/3146378/133583507-20cb7de3-bd97-4dc0-a357-9a63bc95c18d.png">


```suggestion
    const dialCode = COUNTRY_CODES.find(x => dialPhone.match(new RegExp(`^\\${x}|${x.replace('+', '')}`, 'gi')))?.dialCode
```

Reply:
Handling the country code prefix without the `+` sign doesn't seem a good idea. 
For example, you set the country to `+62` (Indonesia) and enter your number in the input box: `81XXX`, then it will change to +81 (Japan) country code without the user realizing it. It also makes sense for the user to define the + sign when defining the country code explicitly.

Reply:
Ah I see, thank you for finding the edge case 🤣